SUSE distros ship a 'wwwrun' user instead of 'www'.  The compile-time default
user *must* exist or slowcgi will not start, even with the -u option.

--- slowcgi.c	2018-07-26 13:27:19.518389330 -0500
+++ slowcgi.c	2018-07-26 13:35:27.667258224 -0500
@@ -44,7 +44,7 @@
 #endif
 
 #define TIMEOUT_DEFAULT		 120
-#define SLOWCGI_USER		 "www"
+#define SLOWCGI_USER		 "wwwrun"
 
 #define FCGI_CONTENT_SIZE	 65535
 #define FCGI_PADDING_SIZE	 255
@@ -270,7 +270,7 @@
 struct slowcgi_proc	slowcgi_proc;
 int			debug = 0;
 int			on = 1;
-char			*fcgi_socket = "/var/www/run/slowcgi.sock";
+char			*fcgi_socket = "/var/lib/wwwrun/slowcgi.sock";
 
 int
 main(int argc, char *argv[])
--- slowcgi.8	2018-08-09 14:40:51.869804185 -0500
+++ slowcgi.8	2018-08-09 14:40:42.369864309 -0500
@@ -47,15 +47,15 @@
 .Pp
 .Nm
 opens a socket at
-.Pa /var/www/run/slowcgi.sock ,
-owned by www:www,
+.Pa /var/lib/wwwrun/slowcgi.sock ,
+owned by wwwrun:www,
 with permissions 0660.
 It will then
 .Xr chroot 8
 to
-.Pa /var/www
+.Pa /var/lib/wwwrun
 and drop privileges to user
-.Qq www .
+.Qq wwwrun .
 .Pp
 The options are as follows:
 .Bl -tag -width Ds
@@ -78,14 +78,14 @@
 .Ar socket .
 .It Fl U Ar user
 change the owner of
-.Pa /var/www/run/slowcgi.sock
+.Pa /var/lib/wwwrun/slowcgi.sock ,
 to
 .Ar user
-and its primary group instead of the default www:www.
+and its primary group instead of the default wwwrun:www.
 .It Fl u Ar user
 Drop privileges to
 .Ar user
-instead of default user www and
+instead of default user wwwrun and
 .Xr chroot 8
 to
 the home directory of
